packages/utils/navigation.ts

```diff
@@ -106,6 +111,11 @@ const navConfig: {
     Governance: "governance",
     UserPublicProfile: "user/:id/:tab?",
     RiotersFooter: "rioters-footer",
+    AdministrationDashboard: "launchpad/admin",
+    LaunchpadApplications: "launchpad/applications",
+    ApplicationReview: "launchpad/applicationReview",
+    ReadyLaunchpadApplications: "launchpad/readyAapplications",
+    AllProjectAdministrationDash: "launchpad/AllAapplications",
```

Comment:
Please can you rename name routes like that:
- `launchpad/application-review`
- `launchpad/ready-applications`
- `launchpad/all-applications`